### ISSUE TITLE
Catch "…" in extension authors and turn into others flag

### ISSUE THREAD

**Issue by jthingelstad** [WikiApiary/WikiApiary#37]
The string "…" in the author array of an extension should indicate "and others". This should probably be a boolean flag applied to the extension?

**Comment by kghbln** [comment]
Right, a good idea though its more for for showing appreciation (very important) than for querying. :+1:

**Comment by apexkid** [comment]
Which file is this issue related to?

**Comment by kghbln** [comment]
"foreground.php". See the referenced pull request.

**Comment by apexkid** [comment]
Thanks for your response kghbin but I don't get why this issue is still opened. I think the pull request referenced here fixes it, doesn't it?

The flag is automatically set in Special:Version like this:

  function listAuthors( $authors, $extName, $extDir ) {
        $hasOthers = false;
        $list = array();
        foreach ( (array)$authors as $item ) {
            if ( $item == '...' ) {
                $hasOthers = true;

Please let me know if I am thinking wrong.

**Comment by kghbln** [comment]
Ah - no this issue is not about the code of extensions but about how to make "and others" visible on extension's pages in WikiAPIary. So far this information gets omitted and this is why the issue is still open. See e.g. [SMW](https://wikiapiary.com/wiki/Extension:Semantic_MediaWiki) on WikiAPIary: "and others" is not shown though the extension says [has this information](https://semantic-mediawiki.org/wiki/Special:Version).

**Comment by mecyborg** [comment]
Why the check condition for "..." will not solve this problem ?
